File: 백준/Bronze/1159. 농구 경기/농구 경기.cc
#include <bits/stdc++.h>
using namespace std;

int cnt[26];
int N;
string s, s2;

int main() {
	cin >> N;
	for (int i = 0; i < N; i++) {
		cin >> s;
		cnt[s[0] - 'a']++;
	}
	
	for (int i = 0; i < 26; i++) {
		if (cnt[i] >= 5) s2 += (i + 'a');
	}

	if (s2.size()) cout << s2 << "\n";
	else cout << "PREDAJA" << "\n";

	return 0;
}

File: 백준/Bronze/1159. 농구 경기/농구 경기.java
import java.io.*;

public class Main {
    private static int[] cnt = new int[30];
    private static int n = 0;

    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        BufferedWriter bw = new BufferedWriter(new OutputStreamWriter(System.out));

        n = Integer.parseInt(br.readLine());
        for (int i = 0; i < n; i++) {
            String name = br.readLine();
            char c = name.charAt(0);
            cnt[c - 'a']++;
        }

        StringBuilder sb = new StringBuilder();
        for (int i = 0; i < cnt.length; i++) {
            if (cnt[i] >= 5) {
                sb.append((char) (i + 'a'));
            }
        }

        if (sb.length() == 0) bw.write("PREDAJA");
        else bw.write(sb.append("\n").toString());

        bw.flush();
        br.close();
        bw.close();
    }
}
